### PR TITLE
allow for images in random order

### LINKED ISSUE #133: Random Picture View
Is it possible to make a config true false for random picture.
so if i have a lot of pictures i allways see them in time where is send. the most time new pictures or old pictures. 
a nice feature is to do this randomly.

### PR DESCRIPTION
**Pull request by call-me-matt**
fixes #133

### CODE REVIEW

File: js/renderer.js [hunk 1]
```diff
@@ -749,8 +766,8 @@ function newImage(sender, type, newImageArray) {
     timer: 5000,
     icon: "success"
   }).then((value) => {
-    currentImageIndex = images.length;
-    loadImage(true, 0);
+    currentImageIndex = 0;
+    loadImage(false, 0, true);
```
Comment (call-me-matt) [inline js/renderer.js:770]: please verify this is ok. Need to jump directly to a new picture, as the other logic won't work in random order.

**Review by gegu — APPROVED** [review]
I think the extension fits.

I merge it into the develop branch. 

We had a problem that the master was before the develop branch, which I fixed.
In the future we should stick to the workflow, otherwise the merges will get complicated.

**Comment by call-me-matt** [comment]
Great! Thank you

File: js/renderer.js (hunk 1 above)
Comment (call-me-matt) [reply js/renderer.js:770]: I think it has the side-effect that when new pictures come in while in pause, the picture changes to the latest one? (To be investigated)

**Pull request merged**